lib/repository.py

```diff
@@ -23,6 +23,8 @@ def download_from_repository(filepath, newpath, repo=REPOSITORY):
         def reporthook(a,b,c):
             print "%3.1f-%s" % (min(100, float(a * b) / c * 100), filename),
             sys.stdout.flush()
+            print
```

Comment:
retriever update results
0.0-bbs.py 85.1-bbs.py 100.0-bbs.py 0.0-EA_del_moral_2010.script 100.0-EA_del_moral_2010.script 0.0-npn.py 100.0-npn.py 0.0-bbs50stop.py 72.7-bbs50stop.py 100.0-bbs50stop.py 0.0-EA_clark2006.py 1
added new live to print each script on a new line (removed the print and include newline in  'print "%3.1f-%s\n" %'